### PR TITLE
added email  in contact section

### LINKED ISSUE #86: Add my email to the contact section
**Is your feature request related to a problem? Please describe.**
Currently, there is no email in contact section

**Describe the solution you'd like**
Add the email icon and put like this "Email (Personal) : yoshithasbrathnayake@gmail.com"

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by muskaan04**
## Purpose
Add the email address with icon in the contact section
The purpose of this Pull Request is to fix #86 

## Goals
To innovate the contact section by adding the email address

## Approach
Creating the tags and using Itag adding email icons and email addresses. 
 
### Preview Link
https://pr-123-NEW-YOSHITHA-WEBSITE.surge.sh

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: dark/assets/css/styles.css [hunk 1]
```diff
@@ -561,6 +561,15 @@ html {
     transition-delay: 1.1s;
 }
 
+.contact-text {
+    text-align: center;
+    font-size: medium;
+    color: black;
+    background-color: var(--secondary-color);
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:568]: Remove the background color its not good @muskaan04

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by muskaan04** [comment]
@YoshithaRathnayake updated the changes

File: dark/index.html [hunk 2]
```diff
@@ -312,7 +313,7 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
       <!-- Contact Section -->
 
       <section id="contact-section">
-        <h1>CONTACT ME</h1>
+      <h1>CONTACT ME</h1>
```
Comment (YoshithaRathnayake) [inline dark/index.html:316]: Remove this title bcs already there is a title @muskaan04

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 2 above)
Comment (muskaan04) [reply dark/index.html]: Changes done @YoshithaRathnayake

**Review by muskaan04 — COMMENTED** [review]
Done

File: dark/index.html [hunk 3]
```diff
@@ -312,7 +313,6 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
       <!-- Contact Section -->
 
       <section id="contact-section">
-        <h1>CONTACT ME</h1>
         <h3>CONTACT <span class="text-info">ME</span></h3>
         <p class="contact-text">📚 Education : St. Anthony's College, Kandy</p>
```
Comment (YoshithaRathnayake) [inline dark/index.html:317]: Remove this also we have another @muskaan04

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 3 above)
Comment (muskaan04) [reply dark/index.html:317]: @YoshithaRathnayake  updated as per specifications

Comment (YoshithaRathnayake) [reply dark/index.html:317]: So can you remove the p tag @muskaan04 
```suggestion
        <p class="contact-text">📚 Education : St. Anthony's College, Kandy</p>

```

Comment (muskaan04) [reply dark/index.html]: Updated @YoshithaRathnayake

File: dark/index.html [hunk 4]
```diff
@@ -326,6 +326,16 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
           </p>
+           <p class="contact-text">
+            <i class="fa fa-envelope"></i>Email (Personal) : yoshithasbrathnayake@gmail.com
+            </p>
+            <br>
+          <p class="contact-text">
+                    📚 Education : St. Anthony's College, Kandy
+            </p>
+            <br>
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:337]: Remove these two unwanted lines @muskaan04

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 5]
```diff
@@ -312,9 +313,8 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
       <!-- Contact Section -->
 
       <section id="contact-section">
-        <h1>CONTACT ME</h1>
         <h3>CONTACT <span class="text-info">ME</span></h3>
-        <p class="contact-text">📚 Education : St. Anthony's College, Kandy</p>
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:317]: Remove this unwanted line also @muskaan04

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 6]
```diff
@@ -326,6 +326,16 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
...
+           <p class="contact-text">
+            <i class="fa fa-envelope"></i>Email (Personal) : yoshithasbrathnayake@gmail.com
+            </p>
+            <br>
+       
+
+
+
+           
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:338]: Remove these unwanted lines also @muskaan04

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 6 above)
Comment (muskaan04) [reply dark/index.html:332]: @YoshithaRathnayake  done

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @muskaan04

**Pull request merged**